tests/test_0057b-virtual-array-numba.py

```diff
@@ -74,8 +74,8 @@ def test_listarray():
         counter = [0]
 
         def materialize():
-            counter[0] += 1
-            return layout
+            counter[0] += 1  # noqa: B023
+            return layout  # noqa: B023
```

Comment:
Safe because `materialize` functions are not collected in a list and used outside the loop. They're only used in the loop.